swift_browser_ui_frontend/src/views/Containers.vue

```diff
@@ -33,12 +33,17 @@
         </option>
       </b-select>
       <div class="control is-flex">
-        <b-switch 
+        <b-switch
```

Comment:
if you want tests to pass this should help:
```
<div class="control is-flex">
        <b-switch
          v-if="bList.length < 500"
          v-model="isPaginated"
        >
          {{ $t('message.table.paginated') }}
        </b-switch>
      </div>
      <div class="control is-flex">
        <b-switch
          v-model="showTags"
        >
          {{ $t('message.table.showTags') }}
        </b-switch>
      </div>
```

but switches can be grouped also as: https://buefy.org/documentation/switch#styles 

however maybe we need to select that switch a tad different than `cy.contains('Paginated').parent().click()` ... maybe add a `data-testid=` attribute to it